themes/theme-docs/src/components/Input.styles.ts

```diff
@@ -23,6 +23,7 @@ export const inputSpacing = {
 // States
 // ---------------
 export const inputFocus = '';
-export const inputDisabled = '';
+export const inputDisabled =
+  'disabled:cursor-not-allowed disabled:text-text-disabled';
```

Comment:
needs also a light grey background so it is easier distinguishable from the an enabled input.

Reply:
please have a loook again

Reply:
nice